apps/www/src/content/docs/installation/nuxt.md

```diff
@@ -13,6 +13,12 @@ Start by creating a new Nuxt project using `create-next-app`:
 npx nuxi@latest init my-app
 ```
 
+### Install TypeScript
+
+
+```bash
+npm install -D typescript
+```
```

Comment:
Can we add some more info why this is needed? It only happened in certain case. I've tested Nuxt locally and it works without `typescript` deps. 

We can do this:

> if you encountered this issue `ERROR  Cannot read properties of undefined (reading 'sys') (x4)`, then installed Typescript as dependency. [Linked issue](https://github.com/nuxt/nuxt/issues/20936)

Reply:
I was actually thinking of this but I thought that it would make the installation section a little bit bloated.

Reply:
I think clearer instruction for installations is important, and it's just a few line, won't add too much bloat 😂